README.md

```diff
@@ -1,15 +1,90 @@
-# 🎨 Cunningham, the best OpenSource Design System
-
-[![CircleCI](https://circleci.com/gh/openfun/cunningham/tree/master.svg?style=svg)](https://circleci.com/gh/openfun/richie/tree/master)
-
+<div align="center">
+  <a href="https://openfun.github.io/cunningham"><img src="./resources/banner.png" alt="" /></a>
+</div>
+<br/>
 <p align="center">
     <a href="https://openfun.github.io/cunningham"><b>📚&nbsp;&nbsp;Documentation</b></a> •
     <a href="https://www.figma.com/file/sSEMhow3g5zpEkyDLEQ1ZO/DS-Cunningham?node-id=47%3A261&t=nKRnadK4sslyVxD7-0"><b>🖌️&nbsp;&nbsp;Figma</b></a>
 </p>
```

Comment:
I'll just keep a level-one header especially for SEO.

```suggestion
<div align="center">

<a href="https://openfun.github.io/cunningham"><img src="./resources/banner.png" alt="" /></a>

# Cunningham

**A design system and a components library.**

<p>
    <a href="https://openfun.github.io/cunningham"><b>📚&nbsp;&nbsp;Documentation</b></a> •
    <a href="https://www.figma.com/file/sSEMhow3g5zpEkyDLEQ1ZO/DS-Cunningham?node-id=47%3A261&t=nKRnadK4sslyVxD7-0"><b>🖌️&nbsp;&nbsp;Figma</b></a>
</p>

</div>
```

WDYT ?
<img width="757" alt="image" src="https://github.com/openfun/cunningham/assets/9265241/1dc614d6-29fc-45b8-b650-36a4b699c84a">